mailersend/__init__.py

```diff
@@ -274,11 +277,28 @@ def send(self, mail_from, mail_to, mail_subject, mail_content, mail_text=None):
             **self.mail_text,
         }
 
-        # print(json.dumps(self.headers_default))
-        # print(json.dumps(message))
+        return self.email_send_request(message)
 
+    def send_mail_with_template(self, mail_from: str, mail_to: list, template_id: str) -> str:
```

Comment:
We shouldn't need 2 distinct functions to send with template or pure HTML content. It would be easier for the user to define something like the following:

```python

# send with pure HTML content
mailer.send(my_mail, recipient_list, subject, html, text, None)

# send with template ID
mailer.send(my_mail, recipient_list, subject, None, None, "<TEMPLATE_ID>")
```